tox.ini

```diff
@@ -9,7 +9,7 @@ deps =
     psycopg2==2.7.3.2
     sqlalchemy==1.3.0
     pytest==3.10.1
-    alembic==0.7.6
+    alembic==1.4.2
```

Comment:
`ColumnComment` was added in Alembic v1.0.6.

I couldn't find a way to specify a minimum version for a package that may or may not be present in `setup.py`. Any suggestions?

I guess in a sense this is a breaking change... should we bump the sqlalchemy-redshift version to 0.8.0?

Alternatively, I could add a try/except to support Alembic version < 1.0.6.

Reply:
Would you be able to wrap the `compiles(ColumnComment, 'redshift')(postgresql.visit_column_comment)` line in a conditional that checks the Alembic version? If so, then we can probably avoid breaking compatibility.

Reply:
Sounds good :+1: 

I can get to this Monday

Reply:
OK should be good to go, PTAL.

I left this `tox.ini` change in place though so that we're testing against a reasonably recent Alembic version. Also we'd need to bump to at least v1.0.6 unless we want to check the version in tests too.